libs/employer/feature/dashboard/src/lib/components/account-access/user-list/user-list.component.ts

```
@@ -183,7 +184,7 @@ export class UserListComponent {
             return this._workApiRoutesService.updateEmployerProfileLinkAccessLevel$(this._userProfile?.id || '', user.userId, userForm.role).pipe(
               tap(() => {
                 this._nuverialSnackBarService.notifyApplicationSuccess();
-                this.pagingRequestModel.pageNumber = 0;
+                this.pagingRequestModel.reset();
```

nit: I know I said to use the reset here, but thinking it over we don't want the user to go to the first page or change the pagination, we just want to refresh the view.

---

Ok no problem, we can still use the reset function except pass in the parameters we want to keep

---

If we are keeping all of them, no need to reset

---

I misunderstood, if we are also not resetting pageNumber I will remove the function calls here then